### ISSUE TITLE
About resume-from from different settings

### ISSUE THREAD

**Issue by anonymoussss** [open-mmlab/mmdetection#3800]
I used 2 GPUs to train a model before，but it was interrupted due to special reasons. Now I am using 4 GPUs and use resume-from to resume training. After I change the learning rate according to the batch size (2 times lr for 4 GPUs),  in the training log, the number of iterations becomes smaller due to the doubled total batch size, but the learning rate in the training log is still the previous setting (for 2GPUs). Can I use resume-from under different GPU conditions?

**Comment by anonymoussss** [comment]
Specifically, I want to know if the model is restored under a different number of GPUs, can I change the learning rate?   If I resume training with a different number of GPUs, will the final performance be worse?

**Comment by xvjiarui** [comment]
Hi @ancientmooner 
The learning rate is also restored from the checkpoint. You may use `load-from` instead.
